inc/namespace.php

```diff
@@ -3,11 +3,16 @@
 namespace Altis\Local_Server;
 
 use function Altis\get_config;
+use function Altis\get_environment_architecture;
 
 /**
  * Configure environment for local server.
  */
 function bootstrap() {
+	if ( get_environment_architecture() === 'local-server' ) {
```

Comment:
Sorry for the back and forth, I wasn't super clear again. This check is already done here:

https://github.com/humanmade/altis-local-server/blob/master/load.php#L10

When a module is registered the `bootstrap` callback is only run if the `enabled` setting is true, so you don't need to check for this again here.

If someone was to manually enable the module in a different context for some reason then presumably they know what they're doing. If we didn't trust them to know what they're doing we'd be better off making this check into a general way to return early eg:

```php
if ( get_environment_architecture() === 'local-server' ) {
     return;
}
```

The rest of the code in the `bootstrap()` function assumes you're running within the docker environment - eg. looking at `getenv()` to set constants and so on.